.github/workflows/build-ci.yml

```
@@ -30,7 +30,11 @@ jobs:
                     - php: "8.1"
                       laravel: "10.*"
                       mongodb: "5.0"
-                      mode: "low-deps"
+                      mode: "--prefer-lowest"
+                    - php: "8.4"
+                      laravel: "11.*"
+                      mongodb: "7.0"
+                      mode: "--ignore-platform-reqs"
```

`ignore-platform-reqs` is a broad sword and would allow installing despite missing extension and other dependencies being wrong. If you want to specifically allow installing on a newer PHP version, you can tell composer to do so by using the `ignore-platform-req` option:

```suggestion
                      mode: "--ignore-platform-req=php+"
```